src/main/java/com/bitalino/comm/BITalinoFrameDecoder.java

```diff
@@ -57,31 +57,28 @@ public static BITalinoFrame decode(final byte[] buffer,
         frame.setDigital(2, (buffer[j - 1] >> 5) & 0x01);
         frame.setDigital(3, (buffer[j - 1] >> 4) & 0x01);
 
-        // parse buffer frame
-        if (totalBytes >= 3)
-          frame
-              .setAnalog(
+        // MODIFIED by sveinpg.
```

Comment:
Please, remove this comment. You'll will become a contributor no matter what after I merge this.